src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+    "$ref": "#/definitions/LPA",
+    "definitions": {
+        "LPA": {
+            "type": "object",
+            "description": "The main schema for the Lasting Power of Attorney Document.",
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "format": "integer",
+                    "description": "The LPA Documents unique identifier.",
+                    "title": "The unique LPA identifier"
+                },
+                "type": {
+                    "type": "string",
+                    "format": "{health_welfare|property_finance}",
+                    "description": "Is this a Health & Welfare or Property & Finance LPA.",
+                    "title": "LPA Type (Health & Welfare or Property & Finance LPA)"
+                },
+                "preferences": {
+                    "type": "string",
+                    "maxLength": 900,
+                    "description": "Any preferences on the LPA.",
+                    "title": "Preferences on the LPA"
+                },
+                "instructions": {
+                    "type": "string",
+                    "maxLength": 900,
+                    "description": "Any instructions on the LPA.",
+                    "title": "Instructions on the LPA"
+                },
+                "donor": {
+                    "$ref": "#/definitions/Donor",
+                    "description": "The Donor on the LPA.",
+                    "title": "Donor details for the LPA"
+                },
+                "witness": {
+                    "$ref": "#/definitions/Witness",
+                    "description": "The Witness for the LPA.",
+                    "title": "Witness on the LPA"
+                },
+                "certificateProvider": {
+                    "$ref": "#/definitions/CertificateProvider",
+                    "description": "The Certificate Provider on the LPA.",
+                    "title": "Certificate Provider on the LPA"
+                },
+                "attorney": {
+                    "type": "array",
+                    "title": "All attorney(s) on the LPA",
+                    "description": "The Attorney(s) on the LPA.",
+                    "items": {
+                        "$ref": "#/definitions/Attorney"
+                    }
+                },
+                "personToNotify": {
+                    "type": "array",
+                    "title": "Person(s) to Notify",
+                    "description": "The Person(s) to Notify on the LPA.",
+                    "items": {
+                        "$ref": "#/definitions/PersonToNotify"
+                    }
+                }
+            },
+            "required": [
+                "donor",
+                "lpaid",
+                "attorney"
+            ],
+            "title": "Lasting Power of Attorney"
+        },
+        "Donor": {
```

Comment:
Donors currently have a field which specifies whether or not they are able to make a physical mark on the document. Are we going to have a digital equivalent of "the donor is unable to sign the LPA"?